app/boards/arm/bdn9/bdn9_rev2.dts

```diff
@@ -43,30 +43,31 @@
         label = "LEFT_ENCODER";
         a-gpios = <&gpioa 4 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
         b-gpios = <&gpioa 8 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-        resolution = <4>;
+        steps = <20>;
         status = "disabled";
     };
     mid_encoder: encoder_mid {
         compatible = "alps,ec11";
         label = "MID_ENCODER";
         a-gpios = <&gpioa 9 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
         b-gpios = <&gpioa 10 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-        resolution = <4>;
+        steps = <20>;
         status = "disabled";
     };
     right_encoder: encoder_right {
         compatible = "alps,ec11";
         label = "RIGHT_ENCODER";
         a-gpios = <&gpioa 15 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
         b-gpios = <&gpiob 3 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-        resolution = <4>;
+        steps = <20>;
         status = "disabled";
     };
 
     sensors: sensors {
         compatible = "zmk,keymap-sensors";
         status = "disabled";
         sensors = <>;
+        triggers-per-rotation = <20>;
```

Comment:
20 steps and 20 triggers per rotation doesn't seem like it would be a correct value for many encoders. Did you mean to set steps to 80 everywhere?

Reply:
I did. Updated.